src/rules.ts

```diff
@@ -1,6 +1,83 @@
-import {
-  edit, noopTest,
-} from './helpers.ts';
+export const noopTest = { exec: () => null } as unknown as RegExp;
+
+export function edit(regex: string | RegExp, opt?: string) {
```

Comment:
```suggestion
export function edit(regex: string | RegExp, opt = '') {
```

Can you set the default here and remove line 5?